### PR TITLE
[docs] revise readme and changelog for OpenYurt v0.6.0

### PR DESCRIPTION
**Pull request by rambohe-ch**
/kind documentation

#### What this PR does / why we need it:
revise readme and changLog for releasing OpenYurt v0.6.0

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/assign @Fei-Guo @gnunu @kadisi @SataQiu 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,78 @@
...
+
+This YurtCluster Operator is to translate a vanilla Kubernetes cluster into an OpenYurt cluster, through a simple API (YurtCluster CRD).
+And we recommend that you do the cluster conversion based on the declarative API of YurtCluster Operator. More information can be
+found at: ([#389](https://github.com/openyurtio/openyurt/pull/389), [#518](https://github.com/openyurtio/openyurt/pull/518), [yurtcluster-operator](https://github.com/openyurtio/yurtcluster-operator), [@SataQiu](https://github.com/SataQiu), [@gnunu](https://github.com/gnunu))
+
+- Yurtctl init/join
+
+In order to improve efficiency of creating OpenYurt cluster, a new tool named [sealer](https://github.com/alibaba/sealer) has
+been integrated into `yurtctl init` command. and OpenYurt cluster image(based on Kubernetes v1.19.7 version) has been prepared.
+Users can use `yurtctl init` command to creating OpenYurt cluster controlPlane, and use `yurtctl join` to add worker nodes(including
```
Comment (SataQiu) [inline CHANGELOG.md:38]:
```suggestion
Users can use `yurtctl init` command to create OpenYurt cluster control-plane, and use `yurtctl join` to add worker nodes(including
```

File: README.md [hunk 2]
```diff
@@ -72,34 +75,22 @@ Please check the [resource and system requirements](./docs/resource-and-system-r
...
 ## Getting started
 
-OpenYurt supports Kubernetes versions up to 1.20. Using higher Kubernetes versions may cause
+OpenYurt supports Kubernetes versions up to 1.21. Using higher Kubernetes versions may cause
 compatibility issues.
 
-You can setup the OpenYurt cluster [manually](docs/tutorial/manually-setup.md), but we recommend to start
-OpenYurt by using the `yurtctl` CLI tool. To quickly build and install `yurtctl`,
-assuming the build system has golang 1.13+ and bash installed, you can simply do the following:
+In order to use OpenYurt conveniently, There are several ways to install OpenYurt cluster and you can choose the way that match your situation.
```
Comment (gnunu) [inline README.md:81]: matches

Comment (rambohe-ch) [reply README.md]: fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (rambohe-ch) [reply CHANGELOG.md]: fixed.

**Pull request merged**